lib/active_merchant/billing/gateways/worldpay.rb

```diff
@@ -781,6 +782,20 @@ def add_shopper(xml, options)
         end
       end
 
+      def add_fraud_sight_data(xml, options)
+        return unless options[:custom_string_fields].is_a?(Hash)
+
+        xml.tag! 'FraudSightData' do
+          xml.tag! 'customStringFields' do
+            options[:custom_string_fields].each do |key, value|
+              # transform custom_string_field_1 into customStringField1, etc.
+              formatted_key = key.to_s.camelize(:lower).to_sym
```

Comment:
This is essentially the same thing that we do for [Orbital line items](https://github.com/activemerchant/active_merchant/blob/85753986c9216b89cbcf5c1fc7e1c3ac3f7f80df/lib/active_merchant/billing/gateways/orbital.rb#L490)

Reply:
Nice, I don't love the naming of `custom_string_field` on their part but this is a good solution!